content/home/home-content.md

```diff
@@ -17,6 +20,14 @@ EPA is overseen by two congressional committees, the House Energy and Commerce C
 
 We want clean air, clean water, and an end to environmental racism and injustice. 
 
-**Environmental Enforcement Watch** is series of networked online workshops that bring people together from different backgrounds to analyze the data from Environmental Protection Agency’s <a href="https://echo.epa.gov/" target=_blank >Enforcement and Compliance History Online</a> (ECHO) database. ECHO data includes reported violations of environmental laws, facility inspections, and enforcement actions (or lack thereof) issued by the EPA and state agencies.
+We believe in the public’s right to know, and that we can collaboratively design public open-source systems to analyze, discuss, and act on environmental data collectively. EPA data is available to the public through the <a href="https://echo.epa.gov/" target=_blank >Enforcement and Compliance History Online</a> (ECHO) database. ECHO data includes reported violations of environmental laws such as the Clean Air Act (CAA), Clean Water Act (CWA), and the Resource Conservation and Recovery Act (RCRA), which manages hazardous waste, as well as data on emissions, facility inspections, and enforcement actions issued by the EPA and state agencies. However, the database can be challenging to use and error-prone. EEW works to make this data more accessible, user-friendly, and collectively actionable through <a href="/events">public workshops</a>, <a href="https://github.com/edgi-govdata-archiving" target=_blank >open-source data tools</a>, and projects like the <a href="/reports">Congressional District Report Cards</a> and <a href="https://envirodatagov.org/more-permission-to-pollute-the-decline-of-epa-enforcement-and-industry-compliance-during-covid/" target=_blank >COVID-19 enforcement report</a>.
+
+<big>Upcoming Events:</big>
```

Comment:
Since this is a heading for a page section, it would be more appropriate to use `#` or `##` instead of `<big>`